staging/cse/windows/provisioningscripts/kubeproxystart.ps1

```diff
@@ -33,8 +70,23 @@ if ($KubeproxyFeatureGates -contains "WinDSR=true") {
     $global:KubeproxyArgList += @("--enable-dsr=true")
 }
 
-if ($KubeproxyFeatureGates.Count -ne 0) {
-    $global:KubeproxyArgList += @("--feature-gates=" + ($KubeproxyFeatureGates -join ","))
+$featureGateArgs = ""
+foreach ($feature in $KubeproxyFeatureGates) {
+    # IPv6DualStack feature gate should not be passed to kube-proxy in >= 1.25.0
+    # https://github.com/kubernetes/kubernetes/blob/ef70d260f3d036fc22b30538576bbf6b36329995/pkg/features/kube_features.go#L945
+    if (($feature -like "IPv6DualStack=*") -and ((Compare-SemanticVersion -Version1 $KubernetesVersion -Version2 "1.25.0") -ge 0)) {
```

Comment:
could you use `([version]$KubernetesVersion).CompareTo([version]"1.25.0") -ge 0`?

Reply:
the problem using the built in `version` type is it default to `-1` for any version categories that aren't there. So comparing "1.25" and "1.25.0" doesn't work. I would expect them to be equal but pwsh thinks "1.25" is less than "1.25.0"

```
PS /home/tyler> ([version]"1.25").CompareTo([version]"1.25.0")
-1
```

This may not matter here because OrchestratorVersion always contains the patch version but I just wanted to be safe.